pkg/apiwatch/resources.go

```diff
@@ -60,7 +72,7 @@ type ObjectNotifier interface {
 // is delayed by a few decaseconds (with Nagling) to support
 // invalidations based on events that merely trigger some process of
 // changing the set of API resources.
-func NewAPIResourceInformer(ctx context.Context, clusterName string, client upstreamdiscovery.DiscoveryInterface, invalidationNotifiers ...ObjectNotifier) (upstreamcache.SharedInformer, Invalidatable) {
+func NewAPIResourceInformer(ctx context.Context, clusterName string, client upstreamdiscovery.DiscoveryInterface, invalidationNotifiers ...ObjectNotifier) (upstreamcache.SharedInformer, APIResourceLister, Invalidatable) {
```

Comment:
Why returning the APIResourceLister if it's not being used ? Do you anticipate usage of this ?